# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: security/manager/ssl/components.conf
# Commit: b45bd4ea0000
# Full Hash: b45bd4ea0000e64e4b159cac01d09519d73166f7
# Author: Dana Keeler <dkeeler@mozilla.com>
# Date: 2022-11-17 09:39:01
# Regressor Bug: 1791633
# File Overlap Count: 1
# Description:
#   Bug 1791633 - separate nsITLSSocketControl from nsITransportSecurityInfo r=necko-reviewers,kershaw,jschanck
#   
#   Depends on D160311
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D160313
# ==============================================================================

diff -r 705439f1d269 -r b45bd4ea0000 security/manager/ssl/components.conf
--- a/security/manager/ssl/components.conf	Wed Nov 16 16:37:29 2022 +0000
+++ b/security/manager/ssl/components.conf	Wed Nov 16 16:37:29 2022 +0000
@@ -95,8 +95,6 @@
         'cid': '{16786594-0296-4471-8096-8f84497ca428}',
         'contract_ids': ['@mozilla.org/security/transportsecurityinfo;1'],
         'type': 'mozilla::psm::TransportSecurityInfo',
-        'legacy_constructor':
-        'mozilla::psm::NSSConstructor<mozilla::psm::TransportSecurityInfo>',
     },
     {
         'cid': '{16955eee-6c48-4152-9309-c42a465138a1}',